test/apps/gups.c

```diff
@@ -322,10 +322,10 @@ SHMEMRandomAccess(void)
   int64_t ProcNumUpdates; /* number of updates per processor */
```

Comment:
Might as well unsign this one too, right?  It's just 4*LocalTableSize and it's what's passed to UpdateTable.